x-pack/test/case_api_integration/basic/tests/cases/sub_cases/find_sub_cases.ts

```diff
@@ -35,227 +48,431 @@ export default ({ getService }: FtrProviderContext): void => {
       await supertest.get(`${getSubCasesUrl('case-id')}/_find`).expect(404);
     });
 
-    // ENABLE_CASE_CONNECTOR: once the case connector feature is completed unskip these tests
-    describe.skip('find_sub_cases', () => {
+    // ENABLE_CASE_CONNECTOR: enable these tests once the case connector feature is completed
+    describe.skip('create case connector', () => {
       let actionID: string;
       before(async () => {
         actionID = await createCaseAction(supertest);
       });
       after(async () => {
         await deleteCaseAction(supertest, actionID);
       });
-      afterEach(async () => {
-        await deleteAllCaseItems(es);
-      });
 
-      it('should not find any sub cases when none exist', async () => {
-        const { body: caseResp }: { body: CaseResponse } = await supertest
-          .post(CASES_URL)
-          .set('kbn-xsrf', 'true')
-          .send(postCollectionReq)
-          .expect(200);
-
-        const { body: findSubCases } = await supertest
-          .get(`${getSubCasesUrl(caseResp.id)}/_find`)
-          .expect(200);
-
-        expect(findSubCases).to.eql({
-          page: 1,
-          per_page: 20,
-          total: 0,
-          subCases: [],
-          count_open_cases: 0,
-          count_closed_cases: 0,
-          count_in_progress_cases: 0,
+      describe('basic find tests', () => {
+        afterEach(async () => {
```

Comment:
Moved the deletion of the sub cases to only the basic tests so they don't delete the sub cases for the pagination describe block.